packages/data-provider/src/utils/types.ts

```diff
@@ -206,15 +214,19 @@ export type ActiveSubmissionResponse = {
  * override 'data' object to contain a summary of records
  */
 export type ActiveSubmissionSummaryResponse = Omit<ActiveSubmissionResponse, 'data'> & {
-	data: Record<string, DataActiveSubmissionSummary>;
+	data: {
+		inserts?: Record<string, DataInsertsActiveSubmissionSummary>;
+		updates?: Record<string, DataUpdatesActiveSubmissionSummary>;
+		deletes?: Record<string, DataDeletesActiveSubmissionSummary>;
+	};
```

Comment:
This is the structure of a submission summary. This is used as a response on 2 GET endpoints